_stbt/core.py

```diff
@@ -2628,14 +2628,15 @@ def _hocr_iterate(hocr):
 
 
 def _hocr_find_phrase(hocr, phrase):
-    l = list(_hocr_iterate(hocr))
-    words_only = [(w, elem) for w, elem in l if w.strip() != u'']
+    words_only = [(w.lower().translate(_ocr_transtab), elem)
+                  for w, elem in _hocr_iterate(hocr) if w.strip() != u'']
+    phrase = [unicode(w).lower().translate(_ocr_transtab) for w in phrase]
```

Comment:
I'm a little worried about this unicode(). Passing in a utf-8 encoded string to match_text would have worked before, but now will raise.